VariablesHomework/VariablesHomework.cs

```diff
@@ -8,50 +8,51 @@ public class Variables
         [TestMethod]
         public void Declaration()
```

Comment:
This assignment is about variables, and this block is for Declarations.

What declarations (and initializations) would you need to add to make the Asserts pass without changing the asserts?

Reply:
Basically, yes. Just rename the variable `a` to `number`, and the original Assert will pass without changing the Assert.

The original asserts are:
```C#
Assert.IsTrue(number == 25); // Make this pass
Assert.IsTrue(boolean == true); // Make this pass
Assert.IsTrue(decimalValue == 99.9); // Make this pass
Assert.IsTrue(floatingDecimalValue == 25.4f); // Make this pass
Assert.IsTrue(words == "I am a programmer now"); // Make this pass
Assert.IsTrue(character == 'z'); // Make this pass
```